data/basics/address.go

```diff
@@ -70,44 +54,45 @@ func UnmarshalChecksumAddress(address string) (Address, error) {
 	}
 	var short Address
 	if len(decoded) < len(short) {
-		return Address{}, fmt.Errorf("decoded bad address: %v", address)
+		return Address{}, fmt.Errorf("decoded bad addr: %s %v", decoded, decoded)
```

Comment:
Should this be `, address, decoded`?

Reply:
I think just address. I'll clean this up tonight.